Assets/MixedRealityToolkit/Utilities/MixedRealityToolkitSceneChecker.cs

```diff
@@ -29,11 +29,23 @@ static MixedRealityToolkitSceneChecker()
 
         private static void SceneOpened(Scene scene, OpenSceneMode mode)
         {
+            // If building, don't perform this check
+            if (BuildPipeline.isBuildingPlayer)
+            {
+                return;
+            }
+
```

Comment:
Nit: Could also move into CheckMixedRealityToolkitScene to handle two internet birds with one internet stone, unless the goal here is to explicitly only handle SceneOpened/NewSceneCreated (i.e. I could imagine in Unity 2019 maybe there's some other hook we listen to and then we have to fix it again by adding this check to that hook too).

Could in this context means you can go either way and this looks good to me as well

Reply:
Done.